src/main/java/Duke.java

```diff
@@ -1,10 +1,200 @@
+            printFileContents(filePath);
+        } catch (FileNotFoundException e) {
+            System.out.println("no file found");
+            File dataDirectory = new File("data");
+            dataDirectory.mkdir();
+            File dukeFile = new File("data/duke.txt");
+            //dukeFile.createNewFile();
+
+        }
+
+
+        String greet = "____________________________________________________________\n" +
+                " Hello! I'm Duke\n" +
+                "____________________________________________________________";
+        String exit = "____________________________________________________________\n" +
+                "Bye. Hope to see you again!\n" +
+                "____________________________________________________________\n";
+        System.out.println(logo);
+        System.out.println(greet);
+        System.out.print(" What can I do for you?\n");
+        System.out.print(line);
+
+        String input;
+        Scanner userInput = new Scanner(System.in);
+        input = userInput.nextLine();
+
+        while (!input.equals("bye")) {
+
+            if (input.contains("todo")) {
+                int dividerPosition = input.indexOf(" ");
+                String t = input.substring(dividerPosition + 1);
+                if(dividerPosition==-1 || t.equals("") || !input.contains(" ")){
+                    System.out.print(line);
+                    System.out.print("I don't quite understand." +  "\n" + "Please enter the command again." + "\n");
+                    System.out.print(line);
+                } else {
+                    tasks.add(new Todo(t));
+                    System.out.print(line + "Got it. I've added this task:" + "\n" + new Todo(t) + "\n" + "Now you have " + tasks.size() + " tasks in the list." + "\n" + line );
+                }
+
+            } else if (input.contains("deadline")) {
+                int dividerPosition = input.indexOf(" ");
+                int dividerPosition_1 = input.indexOf("/");
+                String d = input.substring(dividerPosition + 1, dividerPosition_1);
+                String by = input.substring(dividerPosition_1 + 4);
+                if(dividerPosition==-1 || d.equals("") || !input.contains(" ")){ 
+                    System.out.print(line);
+                    System.out.print("I don't quite understand." +  "\n" + "Please enter the command again." + "\n");
+                    System.out.print(line);
+                } else {
+                    tasks.add(new Deadline(d, by));
+                    System.out.print(line + "Got it. I've added this task:" + "\n" + new Deadline(d,by) + "\n" + "Now you have " + tasks.size() + " tasks in the list." + "\n" + line);
+                }
+
+            } else if (input.contains("event")) {
+                int dividerPosition = input.indexOf(" ");
+                int dividerPosition_1 = input.indexOf("/");
+                String e = input.substring(dividerPosition + 1, dividerPosition_1);
```

Comment:
what does `e` mean?
try to name your variables so that they can be read by others